python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -1236,8 +1276,10 @@ def errFunc(p, x, y):
 
                     count += 1
```

Comment:
Do we not warn if we've reached the iteration threshold and are still finding outliers?  I'm guessing we converge very quickly, so it's possible 2 is always sufficient, but I wanted to check if this was an oversight.

Reply:
We never have, but I can add a warning.